transport/src/main/java/io/netty/channel/WriteBufferWaterMark.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+public class WriteBufferWaterMark {
+
+    private int writeBufferLowWaterMark;
+    private int writeBufferHighWaterMark;
+
+    public WriteBufferWaterMark(int writeBufferLowWaterMark, int writeBufferHighWaterMark) {
+        super();
+        if (writeBufferLowWaterMark < 0) {
+            throw new IllegalArgumentException(
+                    "writeBufferLowWaterMark must be >= 0");
+        }
+        if (writeBufferHighWaterMark < writeBufferLowWaterMark) {
+            throw new IllegalArgumentException(
+                    "writeBufferHighWaterMark cannot be less than " +
+                            "writeBufferLowWaterMark (" + writeBufferLowWaterMark + "): " +
+                            writeBufferHighWaterMark);
+        }
+        this.writeBufferLowWaterMark = writeBufferLowWaterMark;
+        this.writeBufferHighWaterMark = writeBufferHighWaterMark;
+    }
+
+    public int getWriteBufferHighWaterMark() {
+        return writeBufferHighWaterMark;
+    }
+
+    public int getWriteBufferLowWaterMark() {
+        return writeBufferLowWaterMark;
+    }
```

Comment:
For keeping style

> Am 07.02.2016 um 00:24 schrieb fu.jian <notifications@github.com>:
> 
> In transport/src/main/java/io/netty/channel/WriteBufferWaterMark.java:
> 
> > +            throw new IllegalArgumentException(
> > +                    "writeBufferHighWaterMark cannot be less than " +
> > +                            "writeBufferLowWaterMark (" + writeBufferLowWaterMark + "): " +
> > +                            writeBufferHighWaterMark);
> > +        }
> > +        this.writeBufferLowWaterMark = writeBufferLowWaterMark;
> > +        this.writeBufferHighWaterMark = writeBufferHighWaterMark;
> > +    }
> > +
> > +    public int getWriteBufferHighWaterMark() {
> > +        return writeBufferHighWaterMark;
> > +    }
> > +
> > +    public int getWriteBufferLowWaterMark() {
> > +        return writeBufferLowWaterMark;
> > +    }
> @normanmaurer why remove get? is it to keep the style with netty 4.x's style? such as channel() to instead of getChannel()? or it is just that there is no need for get it.
> 
> —
> Reply to this email directly or view it on GitHub.
>